automation_tools/__init__.py

```diff
@@ -1852,9 +1852,9 @@ def copy_ssh_key(from_host, to_host):
 
 
 def sync_capsule_tools_repos_to_upgrade(admin_password=None):
-    """This syncs capsule and sat-tools repos in Satellite server.
+    """This syncs capsule repos in Satellite server.
 
-    Useful for upgrading Capsule and Client in feature.
+    Useful for upgrading Capsule in feature.
```

Comment:
No. Capsule doesn;t requires sat-tools for setup as well as for upgrade.
Refer Point 4 from procedure Procedure 7.1 form Document : https://access.redhat.com/documentation/en-US/Red_Hat_Satellite/6.1/html/Installation_Guide/sect-Red_Hat_Satellite-Installation_Guide-Obtaining_the_Required_Packages_for_the_Capsule_Server.html